nix/tests/lanzaboote.nix

```diff
@@ -240,6 +240,17 @@ in
     '';
   };
 
+  synthesis = mkSecureBootTest {
+    name = "lanzaboote-synthesis";
+    machine = { lib, ... }: {
+      boot.bootspec.enable = lib.mkForce false;
+    };
+    testScript = ''
+      machine.start()
+      print(machine.succeed("bootctl"))
```

Comment:
(I'm unfamiliar with these tests, so feel free to ignore / correct me)

Should this maybe check if the output doesn't contain `Secure Boot: disabled (disabled)` or something? I'm imagining `bootctl` will succeed whether or not secureboot is configured (as it does on my computer), so this isn't really testing anything on its own

Reply:
This tests really only tests if the synthesis works (otherwise it shouldn't boot at all). We have another `basic` test that checks the output of bootctl for the secure boot string. But it also wouldn't hurt if we add it here.